monai/transforms/utils.py

```diff
@@ -2190,7 +2190,7 @@ def distance_transform_edt(
         if return_distances:
             dtype = torch.float64 if float64_distances else torch.float32
             if distances is None:
-                distances = torch.zeros_like(img, dtype=dtype)  # type: ignore
+                distances = torch.zeros_like(img.contiguous(), dtype=dtype)  # type: ignore
```

Comment:
This is going to make a copy of `img` if it isn't contiguous, instead I think we can use `torch.zeros_like(img, memory_format=torch.contiguous_format)`.

Reply:
Looks good to me, I'm not sure if the CI/CD issues are related however.

Reply:
Hi Eric, not related to this PR but for this one: https://github.com/Project-MONAI/MONAI/issues/7616.